api/controllers/Irc_EntryController.js

```diff
@@ -63,23 +63,35 @@ module.exports = {
   },
 
   process_event: function (request_body) {
-    if (request_body.operation === 'check in' || request_body.operation === 'update individual') {
-      return this.processEventDetaineeCreateOrUpdate(request_body);
+    if (request_body.operation === 'check in') {
+      return this.processEventDetaineesCreateOrUpdate(request_body);
+    }
+    if (request_body.operation === 'update individual') {
+      return this.processEventDetaineesCreateOrUpdate(request_body);
     }
     throw new ValidationError('Unknown');
   },
 
 
-  processEventDetaineeCreateOrUpdate: (request_body) => {
-    return Subjects.findOrCreate({
-      person_id: request_body.person_id
+  processEventDetaineesCreateOrUpdate: (request_body) => {
+    var person_id = request_body.cid_id + '_' + request_body.person_id;
```

Comment:
`person_id` should be prefixed with something to do with the centre (name or id) not the cid_id